sw/device/tests/sim_dv/pwrmgr_deep_sleep_all_wake_ups.c

```diff
@@ -78,14 +78,20 @@ typedef struct test_wakeup_sources {
 } test_wakeup_sources_t;
 
 enum {
-  // test wake up input 0 : sysrst_ctrl
+  // Test wake up input 0 : sysrst_ctrl
   kTestSysrstCtrlWakeupSource = 0,
 
-  // test wake up input 1 : adc_ctrl
+  // Ttest wake up input 1 : adc_ctrl
   kTestAdcCtrlWakeupSource = 1,
 
-  // test wake up input 2 : pinmux wakeup detector
+  // Test wake up input 2 : pinmux wakeup detector
```

Comment:
Comments shall be complete sentences. (I.e., start with a capital letter, end with a period.)